tests/test_guests.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+from unittest.mock import patch
+import datetime
+from utils import users as users_utils
+from schemas import guests as guests_schema, requests as requests_schema
+
+
+guest_1_data = {
+    'name': 'name_of_guest',
+    'email': 'name@guest.com',
+    'phone': '1234567890',
+}
+guest_2_data = {
+    'name': 'name_of_guest2',
+    'email': 'nam2e@guest.com',
+    'phone': '98746543210',
+}
+
+
+@pytest.fixture
+def client(client: TestClient):
```

Comment:
you can use different name here to make it more clear: client_no_auth

Reply:
Done